vcs/src/main/java/org/openjdk/skara/vcs/openjdk/OpenJDKTag.java

```diff
@@ -106,6 +109,9 @@ public String version() {
      * @return
      */
     public int buildNum() {
+        if (buildNum == null) {
```

Comment:
Under what conditions will this be null? When using a `*-ga` tag? If so, this seems fine.